pkg/download/download.go

```diff
@@ -0,0 +1,168 @@
+package download
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/api"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/download/jsoncreator"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/download/yamlcreator"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/environment"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/rest"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util/files"
+)
+
+var baseProjectName = "" //for folder naming
+var cont = 0
+
+//GetConfigs Entry point that retrieves the specified configurations from a Dynatrace tenant
+func GetConfigs(environments map[string]environment.Environment, path string, downloadSpecificAPI string) (statusCode int) {
+
+	list, err := getAPIList(downloadSpecificAPI)
+	if err != nil {
+		util.Log.Fatal("The API list contains invalid values. Run monaco command to see the available options\n" + err.Error())
```

Comment:
Isn't `util.Log.Fatal` exiting the program? Please use `util.Log.Error`.

Reply:
fixed